paddle/fluid/inference/tests/infer_ut/test_ernie_text_cls.cc

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "test_suite.h"  // NOLINT
+
+DEFINE_string(modeldir, "", "Directory of the inference model.");
+
+namespace paddle_infer {
+
+template <typename T>
+T cRandom(int min, int max) {
+  unsigned int seed = 100;
+  return (min +
+          static_cast<T>(max * rand_r(&seed) / static_cast<T>(RAND_MAX + 1)));
+}
+
+std::map<std::string, paddle::test::Record> PrepareInput(int batch_size) {
+  // init input data
+  int digit_length = 115;
+  paddle::test::Record input_ids, segment_ids;
+  int input_num = batch_size * digit_length;
+  std::vector<int64_t> input_data(input_num, 1);
+  std::vector<int64_t> segment_data(input_num, 0);
+  srand((unsigned)time(NULL));
+  for (int x = 0; x < input_data.size(); x++) {
+    input_data[x] = cRandom<int>(1, 100);
```

Comment:
This cRandom method seems to always return 1 https://ideone.com/Al9qCY

Reply:
I got a build error on CI so I looked at this
![image](https://user-images.githubusercontent.com/17185347/200854531-87b96bdf-b936-4efe-90a1-ffefd15d2e59.png)